### ISSUE TITLE
[Suggestion] Allow build tool to preview schematics again, regardless of research requirements.

### ISSUE THREAD

**Issue by Tygerial** [ldtteam/minecolonies-features#736]
### Prerequisites 

- [ ] I am running the latest **alpha** version of MineColonies for my Minecraft version.
- [x] I checked the MineColonies/Structurize [wiki](https://wiki.minecolonies.ldtteam.com/) and made sure that my issue is not covered there.
- [x] I made sure that **this change request is not a duplicate of any existing change request.**

### Context 

- Minecraft Version: 1.19.2
- MineColonies Version: 1.0.1116-ALPHA
- Related Mods and their Versions:

## Is your feature request related to a problem?
Hard to plan out you village in early game due to inability to place down hut blocks requiring research.

## Enhancement or Change description
~~Allow hut blocks to be placed down before research for that building is done. Prevent the start of the building process instead.~~
Allow the build tool to preview schematics again like before the new build tool system.

### Notes or related things
Requested change would allow one to plan out their village/city in advance, being able to see where building will be that they don't have unlocked yet. Would not affect the leveling process, only allow for planning ahead and city development.

---
## Viewers

* Add a thumbs-up to the change request to vote for it. This helps the request become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by Talyda** [comment]
Yeah, please bring this feature back. It's really important for anyone that wants to leave space for future buildings!

**Comment by Thodor12** [comment]
For planning purposes it's possible to just use wool to keep the locations of where you want to fit in buildings, I do this all the time.

**Comment by MotionlessTrain** [comment]
The issue is that you can't load the preview of those buildings at all. So you don't know the size and where the hut block is going to be, which makes it very hard to use a temporary block like that

**Comment by Talyda** [comment]
Yeah, it's simply not possible to do that in a normal gameplay now.
On the patreon server, in the tree colony I had to exit, go into a creative world, open the hospital preview, count blocks, make sure I remembered them (cos I am not clever enough to actually remember to write them down) THEN go back on the server, count blocks, leave my markers, and work on.

USED to be able to bring up the preview, make markers. Simple.

Really needs to be fixed

**Comment by Thodor12** [comment]
Changed title/description a bit to reflect that we are looking to change the build tool to allow previews again, like what was possible before.
The initially requested change is likely not going to happen.

**Comment by Tygerial** [comment]
While I find that to be a lesser solution, it works better than what we have now. Hope it makes it in!

**Comment by Talyda** [comment]
But we could place the hut blocks before too. There was just a message saying that you needed to do the research to build the building.
Having the hut block placed also allows the previews to pop up when you are placing buildings around them.
Basically want it to be like it was before the build tool change.

**Comment by Thodor12** [comment]
> But we could place the hut blocks before too. There was just a message saying that you needed to do the research to build the building. Having the hut block placed also allows the previews to pop up when you are placing buildings around them. Basically want it to be like it was before the build tool change.

That was through placing the hut block directly, which has never been a recommended solution. We've always recommend usage of the build tool over direct placement.

**Comment by Talyda** [comment]
No, I could always place the hut block with the tool.
...couldn't I?
*runs off to check now*

**Comment by MotionlessTrain** [comment]
You could, indeed
(Was just typing that 😄)

**Comment by Talyda** [comment]
![image](https://user-images.githubusercontent.com/17323101/207969422-4edb7e36-937f-4d74-a5b2-ae5c10b1b3b8.png)
Yup! You get a message =)

Sorry, been having this discussion since the new tool started, and I started to doubt my memory =P

**Comment by Tygerial** [comment]
> ![image](https://user-images.githubusercontent.com/17323101/207969422-4edb7e36-937f-4d74-a5b2-ae5c10b1b3b8.png) Yup! You get a message =)
> 
> Sorry, been having this discussion since the new tool started, and I started to doubt my memory =P

So we know it WAS possible before, now we need either a way to do it now or a reason it isn't possible anymore.
